src/frontend/src/pages/InfoPage.tsx

```diff
@@ -12,11 +12,49 @@ import ChatIcon from '@mui/icons-material/Chat';
 import ExternalLink from '../components/ExternalLink';
 import PageBlock from '../layouts/PageBlock';
 import PageLayout from '../components/PageLayout';
+import DRCEditModal from './DesignReviewEditModal';
+import { useState } from 'react';
+import {
+  batman,
+  superman,
+  theVisitor,
+  greenlantern,
+  wonderwoman,
+  flash,
+  aquaman,
+  alfred
+} from '../../../backend/tests/test-data/users.test-data';
+import DRCView from './DesignReviewView';
 
 const InfoPage: React.FC = () => {
+  const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+  const handleOpenEditModal = () => setIsEditModalOpen(true);
+  const handleCloseEditModal = () => setIsEditModalOpen(false);
+
+  // We will have to make a call to the backend to get this data
+  // currently hardcoded for testing purposes
+  const usersToAvailabilities = new Map([
+    [superman, [1, 2, 3, 4, 5, 6, 7]],
+    [batman, [2, 3, 4, 5, 6, 7]],
+    [theVisitor, [3, 4, 5, 6, 7]],
+    [greenlantern, [4, 5, 6, 7]],
+    [wonderwoman, [5, 6, 7]],
+    [flash, [6, 7]],
+    [aquaman, [7]],
+    [alfred, [7]]
+  ]);
+
+  // We will have to maker a call to the backend to get this data
+  // currently hardcoded for testing purposes
+  const iconData = new Map<number, string>();
+  iconData.set(5, 'warning');
+  iconData.set(10, 'build');
+  iconData.set(20, 'computer');
+  iconData.set(50, 'electrical');
```

Comment:
I'd put all the test data in a design-review.utils.ts file and import it here. Make sure to add a TODO in each comment (like you've done here, but just add the keyword TODO), so we remember to go back in and remove the fake data